back-end/API/app/services/jobOffer_service.py

```
@@ -1,176 +1,152 @@
-        employer = employer_repo.getEmployer(jobOfferToDelete.employerId)
-        enterprise = enterprise_repo.getEnterprise(employer.enterpriseId) 
-
-        if not current_user.isModerator and current_employer.enterpriseId != enterprise.id:
-            raise PermissionException("Permission denied")
+            enterprise = enterprise_repo.find_by_id(dto.enterpriseId) # Just to throw is it is not found
+            dto.isApproved = True
+            dto.approvedDate = datetime.now(timezone.utc)
+        else:
+            dto.isApproved = None
+            # Quand un employeur crée pour la première fois une offre, on crée aussi son entreprise.
+            if current_user.enterpriseId == None:
```

Ruff (0.15.13)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)